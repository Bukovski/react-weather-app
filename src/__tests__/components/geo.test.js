import React from 'react';
import { render, screen } from '@testing-library/react';
import Geo from "../../components/geo.component";


const data = {
	"cityName": "Stakhanov",
	"date": "Sun, November 07"
}

describe("Geo Component", () => {
	it('should render with valid data', () => {
		render(<Geo data={ data }/>);
		
		expect(screen.getByText('Stakhanov')).toBeInTheDocument();
		expect(screen.getByText('Sun, November 07')).toBeInTheDocument();
	});

});