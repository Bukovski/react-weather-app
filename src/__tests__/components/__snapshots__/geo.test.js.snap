// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Geo Component should render with valid data 1`] = `
<div
  class="geo"
>
  <div
    class="geo__location"
  >
    Stakhanov
  </div>
  <div
    class="geo__date"
  >
    Sun, November 07
  </div>
</div>
`;
